main.js

```diff
@@ -0,0 +1,235 @@
+let cell2 = document.getElementById('cell-2');
+let cell3 = document.getElementById('cell-3');
+let cell4 = document.getElementById('cell-4');
+let cell5 = document.getElementById('cell-5');
+let cell6 = document.getElementById('cell-6');
+let cell7 = document.getElementById('cell-7');
+let cell8 = document.getElementById('cell-8');
+let cellNumber;
+let clock = document.getElementById('time');
+let startButton = document.getElementById('start');
+let resetButton = document.getElementById('reset');
+let player1Button = document.getElementById('p1Button');
+let player2Button = document.getElementById('p2Button');
+let pVPButton = document.getElementById('PvP');
+let pVCButton = document.getElementById('PvC');
+let playerTurn = document.getElementById('playerStatus');
+let playerXName = document.getElementById('playerX');
+let playerOName = document.getElementById('playerO');
+let currentPlayer = playerXName.textContent;
+let seconds = Number(clock.textContent);
+let move = 0;
+let gameOn = false;
+let winner = '';
+let computer = false;
+
+// object to aid computer guess function
+
+let cellLookup = {
+	0: cell0,
+	1: cell1,
+	2: cell2,
+	3: cell3,
+	4: cell4,
+	5: cell5,
+	6: cell6,
+	7: cell7,
+	8: cell8
+};
+//winning combos object
+//rows, columns, diagonals
+let winningCombos = {
+	r1: [ cell0, cell1, cell2 ],
+	r2: [ cell3, cell4, cell5 ],
+	r3: [ cell6, cell7, cell8 ],
+	c1: [ cell0, cell3, cell6 ],
+	c2: [ cell1, cell4, cell7 ],
+	c3: [ cell2, cell5, cell8 ],
+	d1: [ cell0, cell4, cell8 ],
+	d2: [ cell2, cell4, cell6 ]
+};
+
+//button listeners
+//sets player x name
+player1Button.addEventListener('click', () => {
+	playerXName.textContent = names.value;
+	names.value = '';
+});
+//sets player o name
+player2Button.addEventListener('click', () => {
+	playerOName.textContent = names.value;
+	names.value = '';
+});
+//sets up settings for a player vs player match
+pVPButton.addEventListener('click', () => {
+	pVCButton.disabled = true;
+	pVPButton.disabled = true;
+	pVPButton.className = 'selectedButton';
+});
+//sets up setting for a player vs computer match
+pVCButton.addEventListener('click', () => {
+	pVCButton.disabled = true;
+	pVPButton.disabled = true;
+	player2Button.disabled = true;
+	computer = true;
+	playerOName.textContent = 'Computer';
+	pVCButton.className = 'selectedButton';
+});
+
+// start button
+//turns on game, disables button after click, displays text
+startButton.addEventListener('click', () => {
+	gameOn = true;
+	startButton.disabled = true;
+	interval = setInterval(timer, 1000);
+	//events to place mark for clicks in boxes
+	cell0.addEventListener('click', mark);
+	cell1.addEventListener('click', mark);
+	cell2.addEventListener('click', mark);
+	cell3.addEventListener('click', mark);
+	cell4.addEventListener('click', mark);
+	cell5.addEventListener('click', mark);
+	cell6.addEventListener('click', mark);
+	cell7.addEventListener('click', mark);
+	cell8.addEventListener('click', mark);
```

Comment:
You could use your `cellLookup` object here, and iterate over it with a `for...in` loop to add your event listeners to each cell in a more concise manner, and save yourself a few lines of code.